utils/rpms/daos.spec

```diff
@@ -59,7 +59,7 @@ BuildRequires: libabt-devel >= 1.0rc1
 BuildRequires: libjson-c-devel
 BuildRequires: boost-devel
 %endif
-BuildRequires: libpmem-devel >= 1.8, libpmemobj-devel >= 1.8
+BuildRequires: libpmem-devel >= 1.11, libpmemobj-devel >= 1.11
```

Comment:
I guess this libpmem version is begging to be put into a macro.  :-)

Should https://github.com/daos-stack/daos/blob/ff8e12ed33fc06a0f6819a645ec0e3fbe8e42bf4/debian/control#L20 have a minimum value set?

Interesting that the debian build only requires libpmemobj-dev and not also libpmem-dev.

I guess that's because of https://github.com/daos-stack/pmdk/blob/a9813a728262df331fdf886d83348c6c3b5fcfc6/pmdk.spec#L391 which means we can actually drop the `BuildRequires: libpmem-devel …` from this spec.